protoc-go/Dockerfile

```diff
@@ -9,7 +9,6 @@ RUN /src/build.sh
 FROM ghcr.io/safetyculture/protoc:1.12.0
 LABEL maintainer="SafetyCulture <info@safetyculture.io>"
 
-ENV S12_PROTO_VERSION 1.20.0
```

Comment:
was duplicated above

Reply:
This is not duplicated. These are different docker images, so the environment variables will not be carried across. Note the `FROM ghcr.io/safetyculture/protoc:1.12.0`

Reply:
right...